### PR TITLE
Update index.html

### PR DESCRIPTION
**Pull request by DivineRoot**
Add my card information.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -109,6 +109,40 @@ <h2>A project for first-time contributions</h2>
...
+          <a href="https://github.com/DivineRoot" target="_blank">DivineRoot</a>
+        </p>
+        <p class="about">
+          A CS Undergrad trying to learn about OSS ❤
+        </p>
+        <div class="resources">
+          <p>3 Useful Dev Resources</p>
+          <ul>
+            <li>
+
```
Comment (Prajwalprakash3722) [inline index.html:126]: Please remove this blank line

**Review by Prajwalprakash3722 — CHANGES_REQUESTED** [review]
In line number 126 there is a black line, please remove that, Then your code will be merged, All the best.

File: index.html (hunk 1 above)
Comment (DivineRoot) [reply index.html]: I removed the line blank line at line number 126. Please have a look.

**Comment by Prajwalprakash3722** [comment]
Thank you 👍🏼

**Comment by lax0248917** [comment]
Hello @DivineRoot, Thank you for making those necessary changes. Congratulations, we have merged your Pull Request 🎉 , I gotta say I really liked you addition of Witcher 3 music to listen to while coding !!! 🎵 . Best of luck in the future.

**Pull request merged**